src/context/VerifyContext.js

```diff
@@ -102,26 +102,26 @@ export function VerifyProvider(props) {
 
   // EVENT HANDLERS
 
-  const approve = async ({ approveAction, id }) => {
+  const approve = async ({ approveAction, captureImage }) => {
```

Comment:
I'd suggest we remove the `Image` part from `captureImage` and `api.approveCaptureImage`. We're approving the whole capture, not just the image, so the names are a bit confusing.

Reply:
Agreed.  I'll change it.

Reply:
I read the below comment first, but I think this clarifies it.

Reply:
I think it's worth replacing `captureImage` with `capture` throughout Verify to make its meaning clearer.